ssm/ssm/projects/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.contrib import admin
+from model_utils.choices import Choices
+
+from ssm.core.models import BaseModel
+from ssm.users.models import User
+
+STATUS = Choices(('started', 'Started'), ('in progress', 'In progress'), ('completed', 'Completed'))
+ROLES = Choices(('project manager', 'Project manager'), ('developer', 'Developer'), ('tester', 'Tester'),
+                ('business analyst', 'Business analyst'), ('front-end developer', 'Front-end developer'),
+                ('back-end developer', 'Back-end developer'), ('data scientist', 'Data scientist'),
+                ('designer', 'Designer'), ('team lead', 'Team lead'), ('product owner', 'Product owner'))
+
+
+class Project(BaseModel):
+    name = models.CharField('Name', max_length=32, unique=True)
+    description = models.TextField('Description', null=True, blank=True)
+    specification = models.FileField('Specification', null=True, blank=True)
+    customer = models.ForeignKey(User, on_delete=models.DO_NOTHING, verbose_name='Customer', null=True, blank=True)
+    start_date = models.DateField('Start date', null=True, blank=True)
```

Comment:
Let's use `title()` convention for explanation text, so `Start date` should be changed to `Start Date`